pkg/chunker/file/file.go

```diff
@@ -107,7 +118,7 @@ func (c *File) Read(ctx context.Context) <-chan []byte {
 					select {
 					case <-ctx.Done():
 						return
-					case event := <-watcher.Events:
+					case event := <-watcherEvents:
```

Comment:
read from `nil` channel blocks forever, so it works as if watcher is not enabled